controllers/resource/networkPolicyGenerator.go

```diff
@@ -24,48 +36,60 @@ func generateDenyAllPolicy(doguResource *k8sv2.Dogu, dogu *core.Dogu) *netv1.Net
 			},
 		},
 	)
-
 }
 
-func generateDoguDepNetPol(doguResource *k8sv2.Dogu, dogu *core.Dogu, dependencyName string) *netv1.NetworkPolicy {
-	return generateNetPolWithOwner(fmt.Sprintf("%s-dependency-%s", dogu.GetSimpleName(), dependencyName), doguResource, netv1.NetworkPolicySpec{
-		PodSelector: metav1.LabelSelector{
-			MatchLabels: map[string]string{
-				k8sv2.DoguLabelName: dependencyName,
-			},
-		}, PolicyTypes: []netv1.PolicyType{
-			netv1.PolicyTypeIngress,
-		},
-		Ingress: []netv1.NetworkPolicyIngressRule{
-			{
-				From: []netv1.NetworkPolicyPeer{
-					{
-						PodSelector: &metav1.LabelSelector{
-							MatchLabels: map[string]string{
-								k8sv2.DoguLabelName: dogu.GetSimpleName(),
-							},
-						},
-						NamespaceSelector: &metav1.LabelSelector{
-							MatchLabels: map[string]string{
-								"kubernetes.io/metadata.name": doguResource.Namespace,
-							},
-						},
-					},
-				},
-			},
-		},
-	})
-}
+func generateNetPol(doguResource *k8sv2.Dogu, coreDogu *core.Dogu, dependencyName string, netPolType NetPolType) *netv1.NetworkPolicy {
```

Comment:
🚫 **[checkstyle]** <funlen> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Function 'generateNetPol' is too long (70 > 60)
<!-- __reviewdog__:ChA4YWMwOTFhZjRmNThiMjFlEgpjaGVja3N0eWxl -->